x-pack/test/cloud_security_posture_functional/pages/alerts_flyout.ts

```diff
@@ -159,7 +159,7 @@ export default function ({ getPageObjects, getService }: SecurityTelemetryFtrPro
       await expandedFlyoutGraph.setKqlQuery('cannotFindThis');
       await expandedFlyoutGraph.clickOnInvestigateInTimelineButton();
       await timelinePage.ensureTimelineIsOpen();
-      await timelinePage.waitForEvents();
+      await timelinePage.isTimelineEmpty();
```

Comment:
pay attention that `waitForEvents` was asserting that we have results
`isTimelineEmpty` - is not asserting. It returns a boolean.

It seems you miss an assertion here